### DISCUSSION TITLE
Direct binding + dimming -what am I doing wrong?

### DISCUSSION THREAD

**Discussion by fbraad** [Koenkk/zigbee2mqtt#12087] [Devices]
In an effort to make my smart home as reliable as possible (thereby increasing Wife Acceptance Factor), I have moved from separate Hue, Aqara, etc. hubs to Home Assistant and z2m. Its working out great so far, except for direct binding that is.

One goal I really want to accomplish, is to bind my on/off/dim buttons directly to my HUE lightbulbs to ensure that they are controllable even if the coordinator/HASS is unresponsive for some reason.

To do this, I have bought a [zigbee 3.0 button](https://www.zigbee2mqtt.io/devices/550D6001.html#schneider%2520electric-550d6001) that should support direct binding, and tried to bind it directly to the relevant bulb in the z2m frontend.
<img width="1083" alt="z2m1" src="https://user-images.githubusercontent.com/2606661/161932582-3f75c6c7-cb72-4045-835c-ef142cf77050.PNG">

Binding is successful and on-off works fine, but when i try to hold to dim/brighten, the bulb immediately goes from 255 to 1 in brightness or reverse. When looking at the state for the button, it seems that **"brightness_top"** (the value for the top-button-pair of the 4 button switch) increases and decreases as it should on button-hold action, but i also see an **"action_rate": 255** value that i suspect could be the culprit/reason why the light brightness increments directly from 1 to 255 in one step.

<img width="772" alt="z2m2" src="https://user-images.githubusercontent.com/2606661/161933251-42626d58-e575-4ed8-b744-50a112fbf472.PNG">

I have no idea how to manipulate the "action_rate" value, or even if this is my issue, or if im missing some direct reporting binding or other setting that should be configured.

Does anyone have any idea what to do from here?

the bulb im trying to control is a [Philips Hue white E14](https://www.zigbee2mqtt.io/devices/9290020399.html#philips-9290020399), and is only bound to the coordinater, as i understand it should be?
<img width="785" alt="z2m3" src="https://user-images.githubusercontent.com/2606661/161934575-c5a7bb09-0c4f-4c37-9f97-1a4cae620211.PNG">

Thanks :)

**Comment by fbraad** [comment]
For anyone else finding this thread: The issue was a bug in the Wiser 550D6001 button i was using. Updating to the button to fw version 2.9.1 fixed the issue.

**Reply by poizan42** [reply]
@fbraad Could you share how you can update the firmware on the 550D6001? Do you need the Wiser gateway?

**Reply by fbraad** [reply]
Ended up borrowing a wiser gateway to do the firmware upgrades… really hoping someone will include Wiser FW updates in z2m soon.

**Reply by pank** [reply]
@fbraad do you know whether it is possible to bind  wiser battery switches to zigbee groups in z2m? (I'm searching for suitable Fuga switches; Ikea only works with devices, not groups, and friends of hue use a protocol that only works via a hub...)

**Reply by fbraad** [reply]
@pank yep. Have it configured for several switches. Just make sure that you are on firmware: 002.009.001 R. Also remember to push a button on the switch immediately as you bind to wake the device..

**Comment by pank** [comment]
Thanks a lot! It sounds promising!

Last question: from the z2m device overview it looks like there's still 
no ota via z2m. Is that correct?

**Reply by fbraad** [reply]
Yep, unfortunately. But at least they seem to have only ever issued one fw update.. which also makes it hard to implement the feature..

**Comment by pjs514** [comment]
Hi, I have been looking for zigbee fuga switches that look like the traditional LK switches like this one https://www.proshop.dk/Smart-Home/LAB3D-Hvid-Afbryder-tangent/2936054. The problem with this one is that it is based on "friends of hue" and doesn't support direct binding. I guess the 550D6001 is the only option for LK fuga and direct binding?

**Comment by thiesens** [comment]
Hi.
I see when I bind endpoint 21 it will affect both buttons on top, and for endpoint 22, it will affect both at the bottom.

Has anyone tried to get the 4 buttons working independently.
